src/Extend/Routes.php

```diff
@@ -21,37 +21,160 @@ class Routes implements ExtenderInterface
     private $routes = [];
     private $removedRoutes = [];
 
+    /**
+     * @param string $appName: Name of the app (api, forum, admin).
+     */
     public function __construct($appName)
     {
         $this->appName = $appName;
     }
 
-    public function get($path, $name, $handler)
+    /**
+     * Add a GET route.
+     *
+     * @param string $path: The path of the route
+     * @param string $name: The name of the route, must be unique.
+     * @param callable|string $handler: ::class attribute of the controller class, or a closure.
+     *
+     * If the handler is a controller class, it should implement \Psr\Http\Server\RequestHandlerInterface,
+     * or extend one of the Flarum Api controllers:
```

Comment:
Is it really necessary to specify the list of all abstract controllers since they implement the required interface already?

That list might also get outdated in the future. It might be better if that's just explained in the documentation somewhere if it's not already the case?

Reply:
It is too much, I think I'll only mention the `Flarum\Api\Controller` namespace, that's enough.